test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -67,7 +67,7 @@ export function setup() {
 			await app.workbench.configurePythonDialog.waitForPageOneLoaded();
 			await app.workbench.configurePythonDialog.next();
 			await app.workbench.configurePythonDialog.waitForPageTwoLoaded();
-			await app.workbench.configurePythonDialog.install();
+			await app.workbench.configurePythonDialog.install(); // TODO - lewissanchez - need to figure out why .notifications-toasts is failing in _waitForInstallationComplete.
```

Comment:
It would be preferred to make an issue for tracking this instead.

And please sync up with @lucyzhang929 on this - are you planning on working on this yourself?

Reply:
No, this was more of a way for me to be able to quickly locate the problematic query selector that was timing out in case I needed to come back to this later. I will sync up with Lucy on this.

Reply:
Issue to track this: https://github.com/microsoft/azuredatastudio/issues/17017